meta-jsdelivr/recipes-jsdelivr/jsdelivr-configure/files/sshd_config_readonly_jsdelivr

```diff
@@ -124,5 +124,10 @@ Match User logs
   AllowTCPForwarding no
   ForceCommand /bin/bash -c "/usr/bin/docker logs globalping-probe -f "
 
+Match User devlogs
```

Comment:
lets document in readme how this user works and what is used for

Reply:
Sure, I'm just waiting for the probe to finish the tests(should finish in 3/4 days)